autogpt_platform/frontend/src/lib/autogpt-server-api/types.ts

```diff
@@ -212,10 +212,6 @@ export type GraphMeta = {
   output_schema: BlockIOObjectSubSchema;
 };
 
-export type GraphMetaWithRuns = GraphMeta & {
-  executions: GraphExecution[];
-};
-
 export type Graph = GraphMeta & {
```

Comment:
Now this is a mirror
```suggestion
/* Mirror of backend/data/graph.py:Graph */
export type Graph = GraphMeta & {
```